_posts/2024-09-16-introducing-opensearch-2-17.md

```diff
@@ -0,0 +1,109 @@
+
+With OpenSearch 2.17 release, we have added the following enhancements to the Anomaly Detection plugin:
+
+* **Support for remote and multiple data sources**: The Anomaly Detection Dashboards now support remote index selection, allowing you to select indexes from remote clusters in addition to local ones. You can choose any number of remote and local indexes, providing greater flexibility in data analysis.
+* **Advanced imputation methods for missing data**: To address gaps in your data streams, we've introduced [new imputation methods](https://opensearch.org/docs/latest/observing-your-data/ad/index/#setting-an-imputation-option), including filling missing values with zeros, previous values, or custom values. These options help maintain data continuity and improve anomaly detection accuracy in the presence of missing data.
+* **Integration of ML models with domain-specific rules**: We've combined ML models with domain-specific rules to enhance prediction precision. You can now filter anomalies based on the relationship between actual and expected values using ratio thresholds or absolute value thresholds. This allows for more tailored anomaly detection that aligns with your specific business requirements.
+
+
+
+## Ease of use
+
+This release also includes enhancements for better usability.
+
+**Simplify index creation with application-based configuration templates for optimized performance and seamless integration**
+
+OpenSearch 2.17 introduces [application-based configuration templates](https://opensearch.org/docs/latest/im-plugin/index-context/) as an experimental feature. If this feature is enabled, your cluster provides a predefined set of component templates that can be used while creating indexes and index templates based on your use case. These templates are designed to optimize performance for various use cases, such as logs and metrics, and are compatible with the schema defined in the OpenSearch catalog to further ease end-to-end integration with visualization experiences. This feature can be used through the newly available `opensearch-system-templates` plugin.
+
+
+## Cost, performance, and scalability
+
+This release introduces updates to help you improve the stability, availability, and resiliency of your OpenSearch clusters.
+
+**Enhance snapshot scalability and resilience with centralized operations and improved data distribution**
+
+In OpenSearch 2.17, we added multiple optimizations to snapshots, making them more scalable and resilient. For clusters with remote-backed storage enabled, the snapshot operation is now centralized at the cluster manager node, removing the overhead of communication with other nodes in the cluster. For non-remote-backed storage clusters, we added support for distributing data at the configured snapshot store evenly using [hashed prefixes](https://opensearch.org/docs/latest/api-reference/snapshots/create-repository/). This helps avoid throttling issues at the snapshot store.
```

Comment:
```suggestion
In OpenSearch 2.17, we added multiple optimizations to snapshots, making them more scalable and resilient. For clusters with remote-backed storage enabled, the snapshot operation is now centralized on the cluster manager node, removing the overhead of communication with other nodes in the cluster. For non-remote-backed storage clusters, we added support for evenly distributing data to the configured snapshot store using [hashed prefixes](https://opensearch.org/docs/latest/api-reference/snapshots/create-repository/). This helps to avoid snapshot store throttling issues.
```